conf/messages.en

```diff
@@ -470,22 +470,22 @@ reportingPeriod.heading = What reportable period is this reporting notification
 reportingPeriod.p1 = Each reportable period is a calendar year that runs from the 1st of January until the 31st of December.
 reportingPeriod.p2 = This reporting notification will be in place from the chosen period until another notification is added.
 reportingPeriod.hint = This is a year, for example 2024.
-reportingPeriod.checkYourAnswersLabel = The reportable period
+reportingPeriod.checkYourAnswersLabel = Reportable period
 reportingPeriod.change.hidden = the reportable period
 
 notificationAdded.title = Reporting notification added
 notificationAdded.panel.heading = Reporting notification added
-notificationAdded.p1 = A new reporting notification has been successfully added for {0}.
-notificationAdded.p2 = You can now make submissions for this platform operator.
-notificationAdded.p3 = Return to
-notificationAdded.p3.link.text = manage your digital platform reporting
-notificationAdded.p4 = You can also
-notificationAdded.p4.link.text = add another reporting notification
+notificationAdded.p1 = A new reporting notification has been successfully added for {0}. We have sent confirmation email to {1} and {2}.
```

Comment:
Should this be "we have sent *a* confirmation email..." or "confirmation emails"?